Assets/Scripts/Room/PlayerSpawner.cs

```diff
@@ -1,13 +1,15 @@
 using Photon.Pun;
 using UnityEngine;
 using PlayerType = RoleSelectManager.PlayerType;
+using Cinemachine;
 
 public class PlayerSpawner : MonoBehaviour
 {
     public static readonly string PLAYER_PROPERTIES_TYPE_KEY = "PlayerType";
 
     [SerializeField] private GameObject knightPrefab;
     [SerializeField] private GameObject dragonPrefab;
+    [SerializeField] private CinemachineVirtualCamera camera;
```

Comment:
Unity is giving a warning that this variable hides the inherited `Component.camera` member. Let's either rename this or add the new keyword in front of it.